README.md

```diff
@@ -1,4 +1,57 @@
-openmrn
-=======
+# OpenMRN
 
-OpenMRN (Open Model Railroad Network)
\ No newline at end of file
+OpenMRN (Open Model Railroad Network) is a set of software libraries that are designed to make it easier to implement support for the NMRA's
+LCC (Layout Comand Control) bus.
+
+The work to define LCC was done by the [OpenLCB group](http://openlcb.org/), and that's a good place to go in order to learn more about LCC.
+In particular, [Introduction to OpenLCB](http://openlcb.org/about-openlcb/introduction-to-openlcb/). Whenever you see _OpenLCB_, just think
+of this as a public working group that is creating standards that, once approved by the NMRA, become LCC standards. The [OpenLCB & LCC Standards]
+(http://openlcb.org/openlcb-and-lcc-documents/layout-command-control-lcc/) page will give you a good idea of status of the specifications.
+
+## Why OpenMRN
+
+OpenMRN is a set of C++ code that is designed to make it easier to implement support for LCC. This might be in accessory decoders, in a command
+station, in a thorttle, or any other device. The code is designed to be able to run on micro controllers. There are currently a number of
+different 32-bit micro controllers supported. To find the full list, you'll need to check the boards folder.
```

Comment:
To build OpenMRN and its example applications for Mac OS X simply requires that the native gcc/g++ or LLVM compiler be installed on the host system.  Otherwise, it builds exactly like it would for Linux.